src/__tests__/lib/command/util/apps-util.test.ts

```diff
@@ -2,33 +2,36 @@ import { jest } from '@jest/globals'
 
 import { AppResponse, AppsEndpoint, AppType, PagedApp, SmartThingsClient } from '@smartthings/core-sdk'
 
-import { PropertyTableFieldDefinition, Table, TableGenerator, ValueTableFieldDefinition } from '../../../../lib/table-generator.js'
-import { APICommand } from '../../../../lib/command/api-command.js'
 import {
-	chooseOptionsDefaults,
+	PropertyTableFieldDefinition,
+	Table,
+	TableGenerator,
+	ValueTableFieldDefinition,
+} from '../../../../lib/table-generator.js'
+import { stringTranslateToId } from '../../../../lib/command/command-util.js'
+import {
+	createChooseFn,
+	type ChooseFunction,
```

Comment:
I've recently learned about this `type` marker (even though it's been around a while!) and have started using it in the PR but I'm realizing now that I've done so rather inconsistently. I plan to do better in the next PR and then open up a discussion with the team in standup to talk about it.